src/cli/cluster/cluster_manager.ts

```diff
@@ -258,7 +258,7 @@ export class ClusterManager {
     );
 
     const ignorePaths = [
-      /[\\\/](\..*|node_modules|bower_components|public|__[a-z0-9_]+__|coverage)[\\\/]/,
```

Comment:
Why remove `[\\\/]` from the end? Maybe you want it to be `([\\\/]|$)` instead?

Reply:
Good suggestion @spalger 👍